LondonBritishLibrary/orient/BLorient549.xml

```diff
@@ -30,21 +31,98 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 139</idno>
-                            
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
+
+<msItem xml:id="ms_i1">
+<locus from="3r"/>
+<title type="complete" xml:lang="gez" ref="LIT4017Mashafa">Faws manfasāwi</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="51r"/>
+<title type="complete" xml:lang="en" ref="LIT1970Mashaf">The order of celebrating matrimony</title>
+<note>The writing on <locus target="#61v">f. 61v</locus> and <locus target="#62r">62r</locus> has been erased.</note>
```

Comment:
One could now pose our favourite question if the text is still complete :) what do you think, @thea-m , @DenisNosnitsin1970 ?

Reply:
I think this could be the case for Syntaxe du Codex, would you agree?

Reply:
I guess the text is anyway incopmple, as if the folia would have been torn out